src/main/scala/org/broadinstitute/dsde/vault/services/analysis/UpdateService.scala

```diff
@@ -16,7 +16,7 @@ trait UpdateService extends HttpService {
 
   val routes = updateRoute
 
-  @Path("/{ID}/outputs")
+  @Path("/{id}/outputs")
```

Comment:
Interesting that the tests worked with lower-cased variable `id` and uppercased path param `ID`. There must be some spray magic going on here.